pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
 
-The default version of the simulation uses Poissonian input, which is defined
-in the file ``network_params.py`` to excite neuronal populations of the
-microcircuit. If no Poissonian input is provided, DC input is calculated, which
-should approximately compensate the Poissonian input. It is also possible to
-add thalamic stimulation to the microcircuit or drive it with constant DC
-input. This can be defined in the file ``stimulus_params.py``.
+To run the simulation, simply use:
 
-Authors
---------
+.. code-block:: bash
 
-Hendrik Rothe, Hannah Bos, Sacha van Albada
+   python run_microcircuit.py
 
+The output will be saved in the directory ``data``.
 
+The code can be `parallelized <https://nest-simulator.readthedocs.io/en/latest/guides/parallel_computing.html>`_ using OpenMP and MPI, if NEST has been built with these applications.
+The number of threads (per MPI process) can be chosen by adjusting ``local_num_threads`` in ``sim_params.py``.
+The command for running the script with two MPI processes would be:
+
+.. code-block:: bash
+
+   mpirun -n 2 python run_microcircuit.py
+
+Note on Parameters
+##################
+
+By default, the simulation uses external Poissonian input to excite all neuronal populations of the microcircuit, see ``poisson_input': True`` in ``network_params.py``.
+If set to ``False``, the Poissonian input is turned off and compensated approximately by calculated DC input.
+In addition to this ongoing external drive, a thalamic stimulation or a stimulation by an external DC input can be switched on in ``stimulus_params.py`` (the default for both types of stimuli is ``False``).
+
+The default random initialization of membrane voltages in this simulation uses population-specific means and standard deviations to reduce an initial activity burst in the network: ``'V_type': 'optimized'`` in ``network_params.py``.
+Previous implementations used the same mean and standard deviation for all populations which corresponds to setting ``'V_type': 'original'``.
+
+Contributions to this PyNEST Microcircuit
+#########################################
+
+Current communicating author: Johanna Senk
+
+2016: first version implemented by Hendrik Rothe, Hannah Bos and Sacha van Albada
+
+2019: optimized initialization of membrane voltages added by Han-Jia Jiang
+
+2020: revision of code and documentation, and adaptation to NEST3 by Johanna Senk
+
+Citation
+########
+
+If you use this code, we ask you to cite the paper by Potjans and Diesmann [1]_ and the NEST release on Zenodo.
+
+Acknowledgments
+###############
+
+This project has received funding from the European Union Seventh Framework Programme ([FP7/2007-2013]) under grant agreement n° 604102 (Human Brain Project, HBP), and the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreement No. 720270 (Human Brain Project SGA1) and No. 785907 (Human Brain Project SGA2).
```

Comment:
I do not know of any global decision. For the microcircuit, @diesmann, @albada and me thought it would be a good idea to be in line with the [multi-area model](https://github.com/INM-6/multi-area-model/blob/master/README.md).

Reply:
I do not know of any global decision. We (@diesmann, @albada and me) just thought it would be ok here to be in line with the [multi-area-model](https://github.com/INM-6/multi-area-model/blob/master/README.md).